docs/source/install.rst

```diff
@@ -205,3 +121,58 @@ indicates if the commit contained any formatting errors (that are automatically
 	To maintain consitency, we use Black version `22.8.0` (as defined into `.pre-commit-config.yaml`). Make sure to stick to version.  
 
 
+
+
+
+Virtual environment
+==================================
+
+While not mandatory, utilizing a virtual environment when installing disdroDB is Recommended. Using a virtual environment for installing packages provides isolation of dependencies, easier package management, easier maintenance, improved security, and improved development workflow.
+
+
+
+To set up a virtual environment, follow these steps :
+
+* **With venv :**  
+
+	* Windows: Create a virtual environment with venv:
+
+		.. code-block:: bash
+
+		   python -m venv venv
+		   cd venv/Scripts
+		   activate
+		   
+
+	* Mac/Linux: Create a virtual environment with venv:
+
+		.. code-block:: bash
+
+		   virtualenv -p python3 venv
+		   source venv/bin/activate
+
+
+
+* **With Conda:**
+
+	* Create the `disdrodb-dev` (or anay other name) conda environment:
+
+		.. code-block:: bash
+
+			conda create --name disdrodb-dev python=3.9 --no-default-packages
+
+	* Activate the disdrodb conda environment:
+
+		.. code-block:: bash
+
+			conda activate disdrodb-dev
+
+		
+	* Manually add the path of your local copy of disdrodb to the ``PYTHONPATH`` environment variable. 
```

Comment:
This is not required now that `pip install -e .` works correctly.

Reply:
ok done